packages/expo-ui/ios/TextView.swift

```diff
@@ -3,12 +3,16 @@
 import SwiftUI
 import ExpoModulesCore
 
-internal final class TextViewProps: ExpoSwiftUI.ViewProps {
+internal final class TextViewProps: ExpoSwiftUI.ViewProps, CommonViewModifierProps {
+  @Field var fixedSize: Bool?
```

Comment:
(sidenote) I would really like us not to have to repeat these in every component

Reply:
me too. trying much different ways with no lock. the main problem is that the `@Field` property wrapper cannot used in extension or base class. maybe to drop @Field usage to allow common code sharing